app/models/search/collection_indexer.rb

```diff
@@ -0,0 +1,115 @@
+class CollectionIndexer < Indexer
+
+  def self.klass
+    "Collection"
+  end
+
+  def self.mapping
+    {
+      "collection" => {
+        properties: {
+          title: {
+            type: "text",
+            analyzer: "collection_title_analyzer",
+            fields: {
+              exact: {
+                type:     "text",
+                analyzer: "exact_collection_analyzer"
+              },
+              keyword: {
+                type: "keyword",
+                normalizer: "keyword_lowercase"
+              }
+            }
+          },
+          challenge_type: {            
+            type: "keyword",
+            null_value: "NULL" 
+          },
+          name: { type: "text", analyzer: "simple" },
+          description: { type: "text", analyzer: "simple" },
+          collection_type: { type: "keyword" },
+          created_at: { type: "date" }
+        }
+
+      }
+    }
+  end
+
+  def self.settings
+    {
+      analysis: {
+        analyzer: {
+          collection_title_analyzer: {
+            type: "custom",
+            tokenizer: "standard",
+            filter: [
+              "lowercase"
+            ]
+          },
+          exact_collection_analyzer: {
+            type: "custom",
+            tokenizer: "keyword",
+            filter: [
+              "lowercase"
+            ]
+          }
+        },
+        normalizer: {
+          keyword_lowercase: {
+            type: "custom",
+            filter: ["lowercase"]
+          }
+        }
+      }
+    }
+  end
+
+  def document(object)
+    object.as_json(
+      root: false,
+      only: [
+        :id, :name, :title, :description, :parent_id, :collection_type, :created_at
+      ]
+    ).merge(
+      closed: object.closed?,
+      unrevealed: object.unrevealed?,
+      anonymous: object.anonymous?,
+      owner_ids: object.all_owners.pluck(:user_id),
```

Comment:
Just noting that we originally wanted to use pseud ids here and in `moderator_ids`, but I can't think of a reason that's actually necessary -- users' collections pages are indeed for the user, not the pseud. 

However, it might be worth double checking how this interacts with orphaning?